projects/js-packages/script-data/src/utils.ts

```diff
@@ -1,7 +1,8 @@
-import { AdminSiteData } from './types';
+import { AdminSiteData, CurrentUserData } from './types';
 
 /**
  * Values come from https://github.com/Automattic/jetpack/blob/128db0505a27dcdcdef5946d60f443173b2ef6cd/projects/packages/status/src/class-host.php#L240
+ * @type {Array<import('@automattic/jetpack-script-data').AdminSiteData['host']>}
```

Comment:
You don't need this as the TS type annotation is already set on like 7.

```suggestion
```

Reply:
How strange! I don't remember adding that, and it isn't related to what I'm changing. Thanks, I'll remove it.